GraphicEditor/ViewModel/AddBuildingViewModel.cs

```diff
@@ -57,7 +65,38 @@ private void AddBuilding(object obj)
         private void SettingButtonLook(string buildingButtonStyle)
         {
             Button.Style = (Style)myResourceDictionary[buildingButtonStyle];
-            Button.Background = new SolidColorBrush(MyButtonColor);
+            Button.Background = new SolidColorBrush(MyButtonColor); 
+            Button.Content = NameText;
+
+            string content = (string)Button.Tag;
+            string[] split = content.Split(" ");
+
+            // [Lemara98] This needs to be corrected because I didn't find appropriate controller for buildings !!!
```

Comment:
There exists controller for adding new buildings.

Reply:
I made some changes to controller (added some that were missing) and services (added some more complexed functionalities)